### ISSUE TITLE
Please add link to log files

### ISSUE THREAD

**Issue by whimboo** [web-platform-tests/wpt.fyi#560]
When you check the following results for Firefox it is not clear what's wrong. Given the missing details and no link to any log, it makes it impossible for me to get this problem fixed.

https://wpt.fyi/results/webdriver/tests/set_window_rect/set.py?label=experimental

It would be great if there were links to the log files as produced by the currently displayed job results.

**Comment by Hexcles** [comment]
Unfortunately I don't think our BuildBot runner actually preserves the logs? @jugglinmike 

However, given that Taskcluster is looking good and it does have easily linkable logs, I'm not sure if we want to spend much time capturing logs from BuildBot.

But yeah, for Taskcluster, I think it'd be a good idea to store the links to the original task groups which would provide a lot of useful information.

**Comment by jugglinmike** [comment]
Buildbot does indeed persist the logs--[here's one from March](http://builds.wpt.fyi/#/builders/3/builds/1). To report this to wpt.fyi, tasks would need to introspect their own ID's. I don't know if that's supported, but I could look into it.

I think the wpt.fyi feature would be implemented in the same way for Buildbot and Taskcluster. The uploaders just need to be able to include a URL in the results payload. Does that sound right to you. @Hexcles?

**Comment by Hexcles** [comment]
Yes, that's the idea, @jugglinmike . Let me know when you figure out whether it's easy to get the task ID within a task.

**Comment by jugglinmike** [comment]
@Hexcles confirmed that [the necessary API is available in Buildbot](http://docs.buildbot.net/current/developer/cls-build.html?highlight=geturl). We can annotate results with links to the "trigger" build. This is a slightly farther removed than I'd like since it includes tasks for, e.g. Chrome stable and Chrome experimental, but it is the most direct URL we have.
